### ISSUE TITLE
Screenreaders for the 95% of people not on OSX

### ISSUE THREAD

**Issue by TheJaredWilcurt** [teachaccess/tutorial#22]
Your goal with accessibility, and teaching it, should be inclusive.

The browser and OS a user has should not restrict them in learning.

There are many screen reading tools used by the visually impaired and handicapped. JAWS and WindowsEyes are both common options. NVDA is a free and open source option that can be installed or ran as a portable app on Windows. It's a good tool for testing on Windows:

* https://www.nvaccess.org
